### ISSUE TITLE
Add `torrentp` to Conda Forge

### ISSUE THREAD

**Issue by RoyiAvital** [iw4p/torrentp#23]
It would be great to have `torrentp` on Conda Forge.
Any chance to add it?

**Comment by iw4p** [comment]
Hey, I tried but It seems the permission of other 3rd party libraries which are not listed on the conda-forge is needed.
[https://github.com/conda-forge/staged-recipes/pull/26906#issuecomment-2221311310](https://github.com/conda-forge/staged-recipes/pull/26906#issuecomment-2221311310)

**Comment by RoyiAvital** [comment]
Is there a way to approach them?

**Comment by iw4p** [comment]
Can’t you download it using pip on the conda?

**Comment by RoyiAvital** [comment]
It won't play nice with others in case of dependency issues.

**Comment by iw4p** [comment]
Is there any conflict with others? Which packages?

**Comment by RoyiAvital** [comment]
No, I was talking in general about the effect of installing a package using `pip` inside a `conda` environment.
